test/e2e/test_speakers.py

```diff
@@ -35,6 +41,41 @@ def test_話者の情報を取得できる(
         ) == hash_long_string(response.json())
 
 
+def test_話者の情報をURLで取得できる(
+    client: TestClient, snapshot_json: SnapshotAssertion, snapshot: SnapshotAssertion
+) -> None:
+    speakers = _speaker_list_adapter.validate_json(client.get("/speakers").content)
+    for speaker in speakers:
+        speaker_id = speaker.speaker_uuid
+        response = client.get(
+            "/speaker_info",
+            params={"speaker_uuid": speaker_id, "resource_format": "url"},
+        )
+        assert snapshot_json(name=speaker_id) == response.json()
+        speaker_info = SpeakerInfo.model_validate_json(response.content)
+        portrait = client.get(speaker_info.portrait)
+        assert portrait.status_code == 200
+        assert snapshot(name=f"{speaker_id}_portrait") == _hash_bytes(portrait.content)
+        for style in speaker_info.style_infos:
+            icon = client.get(style.icon)
+            assert icon.status_code == 200
```

Comment:
ちょっと密度が気になりますね！！

論理的なコードなので、たぶん雑にこういうルールに従うだけでも読みやすくなったりするかもです。

* ブロックが終わったら改行（ifとか）
* 文のあとに文が来ない場合は改行（breakとかassertとか、hoge.fuga()とか。最後のは文じゃないけど。）

それだけでこうなるはず。
```suggestion
    for speaker in speakers:
        speaker_id = speaker.speaker_uuid
        response = client.get(
            "/speaker_info",
            params={"speaker_uuid": speaker_id, "resource_format": "url"},
        )
        assert snapshot_json(name=speaker_id) == response.json()

        speaker_info = SpeakerInfo.model_validate_json(response.content)
        portrait = client.get(speaker_info.portrait)
        assert portrait.status_code == 200
        assert snapshot(name=f"{speaker_id}_portrait") == _hash_bytes(portrait.content)

        for style in speaker_info.style_infos:
            icon = client.get(style.icon)
            assert icon.status_code == 200
```